### ISSUE TITLE
The Dogecoin parser finishes work ahead of time.

### ISSUE THREAD

**Issue by Napulsnik** [gcarq/rusty-blockparser#75]
Hello. The Dogecoin parser finishes work ahead of time.

```
...
[14:32:16] INFO - parser: Status: 1712245 Blocks processed. (left: 2014821, avg:  2140 blocks/sec)
[14:32:26] INFO - parser: Status: 1738425 Blocks processed. (left: 1988641, avg:  2146 blocks/sec)
[14:32:36] INFO - parser: Status: 1764659 Blocks processed. (left: 1962407, avg:  2152 blocks/sec)
[14:32:49] INFO - parser: Done. Processed 1788760 blocks in 13.88 minutes. (avg:  2149 blocks/sec)
[14:32:52] INFO - callback: Done.
Dumped 952803 addresses.
[14:32:52] INFO - main: Fin.
```

**Comment by bkulcsar** [comment]
Same here, and also --start and --end paramters not working.

**Comment by gcarq** [comment]
Is it the same block where it stops? Please add the command how you started it and some relevant log lines so that it can be reproduced easier.

**Comment by bkulcsar** [comment]
Thanks for the fast response!

I started it with the following command: ./rusty-blockparser -c dogecoin -d /mnt/d/Dogecoin/blocks balances
It stoped at block: 1788760
In the logs same as Napulsnik's,  no errors just:
[09:42:36] INFO - parser: Status: 1788760 Blocks processed. (left: 2854972, avg:  313 blocks/sec)
[09:42:49] INFO - parser: Done. Processed 1788760 blocks in 54.64 minutes. (avg:  313 blocks/sec)
[09:42:52] INFO - callback: Done.

I also tried to start with:
 ./rusty-blockparser -c dogecoin -d /mnt/d/Dogecoin/blocks -s 1788760  -e 4643736 balances

But it finished way more faster and only processed 10,000 blocks, again no errors.
